static/app/views/dashboardsV2/detail.tsx

```diff
@@ -724,3 +697,49 @@ const StyledPageContent = styled(PageContent)`
 `;
 
 export default withApi(withOrganization(DashboardDetail));
+
+/**
+ * Reads the layout from an array of widgets.
+ */
+function getDashboardLayout(widgets: Widget[]): RGLLayout[] {
+  return widgets
+    .filter(({layout}) => !!layout)
```

Comment:
Are you checking here to see if layout is defined? can we use `defined` from `sentry/utils` keep it more explicit? And this is just because layout is currently optional on the Widget right? ie there shouldn't be a case where some widgets on a dashboard have a layout and others don't?